bib_rights.pm

```diff
@@ -80,20 +80,16 @@ sub new {
   my $us_fed_pub_exceptions = {};
   $ENV{'us_fed_pub_exception_file'} and do { # us fed pub exception file (file of oclc number of records that shouldn't be considered us fed docs, regardless of 008 coding)
     my $us_fed_pub_exception_file = $ENV{'us_fed_pub_exception_file'};
-    if (-e $us_fed_pub_exception_file) {
-      # uncoverable branch true
-      open (US_FED_PUB_EXCEPTIONS, "<$us_fed_pub_exception_file") or die "can't open $us_fed_pub_exception_file for input: $!\n";
-      print STDERR "using $us_fed_pub_exception_file for us fed pub exceptions\n";
-      my $exception_count = 0;
-      while (<US_FED_PUB_EXCEPTIONS>) {
-        chomp();
-        $us_fed_pub_exceptions->{$_}++;
-        $exception_count++;
-      }
-      print STDERR "$exception_count oclc numbers in us fed pub exception list\n";
-    } else {
-      print STDERR "us_fed_pub_exception_file set to $us_fed_pub_exception_file, not readable\n";
+    # uncoverable branch true
+    open (US_FED_PUB_EXCEPTIONS, "<$us_fed_pub_exception_file") or die "can't open $us_fed_pub_exception_file for input: $!\n";
+    print STDERR "using $us_fed_pub_exception_file for us fed pub exceptions\n";
+    my $exception_count = 0;
+    while (<US_FED_PUB_EXCEPTIONS>) {
+      chomp();
+      $us_fed_pub_exceptions->{$_}++;
+      $exception_count++;
     }
+    print STDERR "$exception_count oclc numbers in us fed pub exception list\n";
```

Comment:
I think this is one of the places where bib_rights is unnecessarily chatty. Might be worth considering just setting `$self->{us_fed_pub_exceptions_size} = $exception_count` and if the client needs this info it can be looked up rather than extracted from STDERR. The same holds true of the other print to STDERR on line 85.

Reply:
Agreed. I will address.

Reply:
@moseshll  see new commit